demisto_sdk/commands/validate/README.md

```diff
@@ -1,154 +1,100 @@
 ## Validate
 
-Makes sure your content repository files are in order and have valid yml file scheme.
-
-**Notes**
-
-In order to run the README validator:
-- Node should be installed on you machine
-- The modules '@mdx-js/mdx', 'fs-extra', 'commander' should be installed in node-modules folder.
-    If not installed, the validator will print a warning with the relevant module that is missing.
-    please install it using "npm install *missing_module_name*"
-- 'DEMISTO_README_VALIDATION' environment variable should be set to True.
-    To set the environment variables, run the following shell commands:
-    export DEMISTO_README_VALIDATION=True
-
-In case of a private repo and an un-configured 'DEMISTO_SDK_GITHUB_TOKEN' or 'DEMISTO_SDK_GITLAB_TOKEN' validation of version bumps in files will be done with the local remote git branch.
+Makes sure your content repository files are in order and have valid file scheme.
 
 **Use Cases**
-This command is used to make sure that the content repo files are valid and are able to be processed by Demisto.
-This is used in our validation process both locally and in Circle CI.
+This command is used to make sure that the content repo files are valid and are able to be processed by the platform.
+This is used in our validation process both locally and in gitlab.
 
 **Arguments**:
-* **--no-backward-comp**
-Whether to check backward compatibility or not.
-* **--no-conf-json**
-Skip conf.json validation.
-* **-s, --id-set**
-Perform validations using the id_set file.
-* **-idp, --id-set-path**
-The path of the id-set.json used for validations.
-* **--prev-ver**
-Previous branch or SHA1 commit to run checks against.
 * **-g, --use-git**
 Validate changes using git - this will check the current branch's changes against origin/master.
 If the **--post-commit** flag is supplied: validation will run only on the current branch's changed files that have been committed.
 If the **--post-commit** flag is not supplied: validation will run on all changed files in the current branch, both committed and not committed.
-* **-pc, --post-commit**
```

Comment:
all these flags still work in the old validate flow right? We should for the time being still have docs for them, maybe under a "deprecated" section or in another file that we link to from here

Reply:
how about renaming the old readme to old_Validate_readme.md with all the info related to the old validate?

Reply:
Will add a new old_validate_readme.md file and will link it to the readme.md